ui/lib/pki/addon/components/page/pki-overview.hbs

```diff
@@ -64,15 +64,14 @@
         @fallbackComponent="input-search"
         data-test-view-certificate-input
       />
-      <button
-        type="button"
-        class="button is-secondary {{unless this.certificateValue 'has-left-margin-s'}}"
+      <Hds::Button
+        @text="View"
+        @color="secondary"
+        class="has-left-margin-s align-self-baseline"
```

Comment:
I think the conditional should still be here?

Reply:
With or without the certificate selected, it acts the same with the conditional or without it. And it pushes the button to next line, which I actually think is OK because only one certificate will ever be selected. 

**Without a conditional** 
![image](https://github.com/hashicorp/vault/assets/6618863/6a448768-696a-459c-9c73-ad741e23c4c5)
![image](https://github.com/hashicorp/vault/assets/6618863/4bac5b04-c1b5-4e9c-9f3c-58fb3f9c0a66)

**With a conditional**
![image](https://github.com/hashicorp/vault/assets/6618863/343fc118-950f-4375-9ff8-21dd7d879807)
![image](https://github.com/hashicorp/vault/assets/6618863/ed731c8b-7448-4213-b485-dbff435bd1e3)